src/openzaak/components/catalogi/models/statustype.py

```diff
@@ -87,19 +86,15 @@ class Meta:
         unique_together = ("zaaktype", "statustypevolgnummer")
         verbose_name = _("Statustype")
         verbose_name_plural = _("Statustypen")
+        indexes = [models.Index(fields=["zaaktype", "-statustypevolgnummer"])]
```

Comment:
There's now two indexes on the same columns:
`unique_together = ("zaaktype", "statustypevolgnummer")` and `indexes = [models.Index(fields=["zaaktype", "-statustypevolgnummer"])]`

Excessive :)